website/archiver/__init__.py

```diff
@@ -26,11 +26,10 @@ class StatResult(object):
     """
     num_files = 1
 
-    def __init__(self, target_id, target_name, disk_usage=0, meta=None):
+    def __init__(self, target_id, target_name, disk_usage=0):
         self.target_id = target_id
         self.target_name = target_name
         self.disk_usage = float(disk_usage)
-        self.meta = meta
 
     def __str__(self):
         return json.dumps(self._to_dict())
```

Comment:
Why not just `str(self._to_dict())`. That way values like `True` and `False` won't get converted to `true` and `false`.